sdks/python/apache_beam/ml/inference/tensorrt_inference.py

```diff
@@ -272,7 +273,8 @@ def run_inference(
 
       return [
           PredictionResult(
-              x, [prediction[idx] for prediction in cpu_allocations]) for idx,
+              x if not drop_example else None,
+              [prediction[idx] for prediction in cpu_allocations]) for idx,
```

Comment:
Can we use `_convert_to_result` here too? I don't think there's a reason these predictions can't be dictionaries, so this will handle that case automatically